poc_iot_verifier/src/runner.rs

```diff
@@ -29,7 +29,8 @@ use tokio::time;
 const DB_POLL_TIME: time::Duration = time::Duration::from_secs(90);
 const LOADER_WORKERS: usize = 10;
 const LOADER_DB_POOL_SIZE: usize = 2 * LOADER_WORKERS;
-const BEACON_INTERVAL: i64 = 60; //minutes
+/// the cadence in minutes at which hotspots are permitted to beacon
+const BEACON_INTERVAL: i64 = 60; //TODO: determine sane value
```

Comment:
we should standardize on seconds to avoid tripping ourselves up later on different values of durations here and there

Reply:
not sure about this one, the check on line 172 is an explicit check in minutes and reducing beacon interval to seconds makes it more difficult to convey the duration ( ie if we change the interval to every 8 hours )

Reply:
Whichever is used - it's best to include the units in the name, like:
- `BEACON_INTERVAL_MINUTES`
- `BEACON_INTERVAL_SECONDS`
- etc

Reply:
yeah good suggestion

Reply:
applied @xandkar's suggestion here: 4e0d8f4949e4b9742da9518ecb9870665ff916b0

Reply:
When possible use Duration instead which adds _types_ instead of names and are perfectly fine being defined as consts

Reply:
it depends on the Duration you're using; a `std::time::Duration` can be defined as a const but a `chrono::Duration` needs to be in a lazy_static block to do the same

Reply:
fixed here: 24731f41f66861269b324a5fbe5a29ae792f50e8